rsvim_core/src/state/fsm/command_line_ex_tests.rs

```diff
@@ -785,10 +793,8 @@ mod tests_confirm_ex_command_and_goto_normal_mode {
     {
       let cmdline_content = stateful._goto_normal_mode_impl(&data_access);
       info!("cmdline content:{cmdline_content:?}");
-      assert_eq!(
-        "Bye1 Bye2 Bye3 Bye4 Bye5 Bye6 Bye7",
-        cmdline_content.as_str()
-      );
+      // After go to normal mode, content is cleared
+      assert_eq!("", cmdline_content.as_str());
```

Comment:
The "_goto_normal_mode_impl" actually returns user's input text contents. When user go back to normal mode (by pressing 'Esc' or 'Enter'), what user has input in the widget is returned.